net/tunneldigger/Makefile

```diff
@@ -1,10 +1,10 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=tunneldigger
-PKG_RELEASE:=1
+PKG_RELEASE:=2
 
 PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=git://github.com/wlanslovenija/tunneldigger.git
+PKG_SOURCE_URL:=https://github.com/wlanslovenija/tunneldigger
```

Comment:
While it doesn't matter too much, canonically these URLs should keep the `.git` suffix when used as a clone URL rather than a site you visit in your browser.